router/core/graphql_prehandler.go

```diff
@@ -142,42 +150,81 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 			r = r.WithContext(resolve.SetTraceStart(r.Context(), traceOptions.EnablePredictableDebugTimings))
 		}
 
-		validatedReq, err := h.accessController.Access(w, r)
-		if err != nil {
-			hasRequestError = true
-			requestLogger.Error(err.Error())
-			writeRequestErrors(r, http.StatusUnauthorized, graphql.RequestErrorsFromError(err), w, requestLogger)
-			return
+		// If we have authenticators, we try to authenticate the request
+		if len(h.accessController.authenticators) > 0 {
+			authenticateSpanCtx, authenticateSpan := h.tracer.Start(r.Context(), "Authenticate",
+				trace.WithSpanKind(trace.SpanKindServer),
```

Comment:
Does it make sense for this to be a `server` span? Based on their [docs](https://opentelemetry.io/docs/concepts/signals/traces/#server):

> A server span represents a synchronous incoming remote call such as an incoming HTTP request or remote procedure call.

It doesn't seem appropriate given, if anything, it would be making HTTP requests so `client` might make sense since it can't be `internal` due to crossing process boundaries?